test/addons-napi/7_factory_wrap/myobject.h

```diff
@@ -1,7 +1,7 @@
 #ifndef TEST_ADDONS_NAPI_7_FACTORY_WRAP_MYOBJECT_H_
 #define TEST_ADDONS_NAPI_7_FACTORY_WRAP_MYOBJECT_H_
 
-#include <node_api.h>
+#include <js_native_api.h>
```

Comment:
Just wondering about changing this since it means that we then have to add node_api.h to the cc file?  Since it is still not going to be able to compile in an alternate implementation I'm wondering why it makes sense to change it?

Reply:
Ok got it. It makes a first step towards breaking the dependency even though it does not go all the way.  A next step would be to extract that bit out of the test files into a single place in the test suite so that it would be easier to change globally as opposed to having to edit every file.  But that is out of scope for this PR so I'm ok with it as is.